octobot_trading/exchanges/connectors/ccxt/ccxt_connector.py

```diff
@@ -123,7 +123,13 @@ async def _load_markets(self, client, reload: bool):
         """
         Override if necessary
         """
-        await client.load_markets(reload=reload)
+        try:
+            await client.load_markets(reload=reload)
+        except Exception as err:
+            # ensure this is not a proxy error, raise dedicated error if it is
+            if proxy_error := ccxt_client_util.get_proxy_error_if_any(self, err):
+                raise octobot_trading.errors.ExchangeProxyError(proxy_error) from err
```

Comment:
requried to identify proxy errors when load_markets makes authenticated requests